README.md

```diff
@@ -1,9 +1,39 @@
 # TypeOfNaN JavaScript Quizzes
 
+![TypeOfNaN Logo javascript quizzes](typeOfNaN-logo.jpg)
```

Comment:
Can we change to:

`![TypeOfNaN JavaScript Quizzes](typeOfNaN-logo.jpg 'TypeOfNaN JavaScript Quizzes')`

Reply:
Also, is it possible to make the logo a bit smaller? Maybe between 50-75% width of the readme file?

Reply:
Sure, i have the file already ready, but need to create another pull request to update the file